### DISCUSSION TITLE
Getting 'IOKit/IOKitLib.h' file not found when building tinygo from source

### DISCUSSION THREAD

**Discussion by shihanng** [tinygo-org/tinygo#3971] [Q&A]
I am new to the project. I want to build tinygo from the source (via the `release` branch), but I am getting the following error message when running the `make` command as instructed by the [documentation](https://tinygo.org/docs/guides/build/#building-tinygo-1).

```
$ make
...
# go.bug.st/serial/enumerator
../../../../../go/pkg/mod/go.bug.st/serial@v1.6.0/enumerator/usb_darwin.go:10:11: fatal error: 'IOKit/IOKitLib.h' file not found
 #include <IOKit/IOKitLib.h>
          ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [Makefile:284: tinygo] Error 1
```

I can find `IOKitLib.h` in the SDK path on my machine.

```
ls -l $(xcrun --sdk macosx --show-sdk-path)/System/Library/Frameworks/IOKit.framework/Versions/A/Headers
...
.rw-r--r-- 5.0k root 18 Aug 13:33 IODataQueueShared.h
.rw-r--r--  15k root 18 Aug 13:33 IOKitKeys.h
.rw-r--r--  80k root 11 Aug 14:17 IOKitLib.h
...
```

Do you have any suggestions on how to solve this issue? Thank you 🙏

**Comment by Gustavomurta** [comment]
I found this file in the package
_https://pkg.go.dev/go.bug.st/serial@v1.6.1/enumerator_
_https://github.com/bugst/go-serial_

But in this file  **usb_darwin.go**, this include is only cited for reference.

https://github.com/bugst/go-serial/blob/master/enumerator/usb_darwin.go

package enumerator

// #cgo LDFLAGS: -framework CoreFoundation -framework IOKit
// #include <IOKit/IOKitLib.h>
// #include <CoreFoundation/CoreFoundation.h>
// #include <stdlib.h>
